tools/VscPreviewExtension/README.md

```diff
@@ -30,7 +30,7 @@ For further information and details about DocFX Flavored Markdown, please refere
 | `ctrl+k q` | `Open Dfm Preview to the side` | Preview to side |
 |  | `show Dfm Show` | Show Source |
 
-  ![PreviewToside](/tools/VscPreviewExtension/images/previewToSide.gif)
+  ![PreviewToside](https://github.com/dotnet/docfx/raw/master/tools/VscPreviewExtension/images/previewToSide.gif)
```

Comment:
Do you have any idea about why it doesn't work in extension homepage while using relative path?